graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -176,6 +176,7 @@ private Map<String, Object> runQueryWithLowLevelClient(String query) throws Atla
 
     private DirectIndexQueryResult performAsyncDirectIndexQuery(SearchParams searchParams) throws AtlasBaseException, IOException {
         AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("performAsyncDirectIndexQuery");
+        AtlasPerfMetrics.MetricRecorder metricSearchTimeout = RequestContext.get().startMetricRecord("asyncDirectIndexQueryTimeout");
```

Comment:
Extra metric metricSearchTimeout not needed, collect the metric in finally and it will give us time for this in metric.

Reply:
Discussed with @aarshi0301 , we can keep it.